### ISSUE TITLE
Permalink text is misleading and links should be moved outside of the heading element

### ISSUE THREAD

**Issue by halindrome** [w3c/respec#827]
(Note: This is related to issue #425).

A COGA reviewer noted that the title text for permalinks is misleading and that the term "permalink" may not be in common usage.  The suggested "Bookmark to [section]" instead of "Permalink for [section]".  

They also noted that ATs, when reading the heading, are going to read out the permalink title as well because it is embedded in the heading element.  So they too have requested that the permalink be moved outside of the heading. 

I am going to move this up on my priority list.  TBH I had forgotten about the issue.

**Comment by marcoscaceres** [comment]
closed via https://github.com/w3c/respec/pull/1822

I'm going to deprecate the old code.
